images/linux/scripts/installers/1604/kubernetes-tools.sh

```diff
@@ -9,10 +9,10 @@ source $HELPER_SCRIPTS/document.sh
 source $HELPER_SCRIPTS/apt.sh
 
 ## Install kubectl
```

Comment:
What is the difference between kubernetes-tools.sh? Could we combine into one?

Reply:
As I see, it uses different deb packages for 16.04 and 18.04:
16.04: `deb https://apt.kubernetes.io/ kubernetes-$(lsb_release -cs) main`
18.04: `deb https://apt.kubernetes.io/ kubernetes-xenial main`

Reply:
16.04: echo $(lsb_release -cs) = xenial
18.04: echo $(lsb_release -cs) = bionic

I suppose we can combine if only string is different

Reply:
You can use these helpers: https://github.com/actions/virtual-environments/blob/master/images/linux/scripts/helpers/os.sh

Reply:
@maxim-lobanov , I believe we can use `xenial` for both OS versions.
@al-cheb , done, thank you!